packages/insomnia-inso/src/commands/run-tests.ts

```diff
@@ -49,7 +49,7 @@ const createTestSuite = (dbSuite: UnitTestSuite, dbTests: UnitTest[]): TestSuite
 export async function runInsomniaTests(
   identifier: string | null | undefined,
   options: Partial<RunTestsOptions>,
-) {
+): Promise<boolean> {
```

Comment:
This type was already here, was this a mistake?

![Screenshot_20210513_080153](https://user-images.githubusercontent.com/15232461/118122899-7d77df00-b3c1-11eb-9f38-4488d691e1d4.png)

```suggestion
) {
```

Reply:
It must have been TSC... because it was presenting me an error here, but isn't now after I just removed it locally: https://github.com/Kong/insomnia/blob/1d0f6faf425ef9d9e94f442756127948d91a1fe2/packages/insomnia-inso/src/cli.ts#L68 😞

Reply:
cool cool, glad I asked.  it's all good :)